auxiliary/RewardsDistributor/foundry.toml

```
@@ -1,7 +1,7 @@
 [profile.default]
 src = "src"
 out = "out"
-libs = []
+libs = ["lib"]
```

This is unnecessary as we only use node_modules and not using git submodules.

```suggestion
libs = []
```